index.html

```diff
@@ -10,20 +10,29 @@
 </head>
 <body>
     <div class="portfolio">
+        <div class="menuModel" id="model" >
+            <span class="close" id="close" >&times;</span>
+            <ul>
+                <li id="menuItem1" > <a href="#landingPage"> Portfolio </a> </li>
+                <li id="menuItem2" > <a href="#about"> About</a> </li>
+                <li id="menuItem3" > <a href="#contact"> Contact </a> </li>
+            </ul>
+        </div>
         <header>
             <nav class="menuNav" >
                 <a href="#" class="logo">Shayan</a>
                 <ul class="mobileMenu" >
-                    <li><img src="./images/menu.svg" class="menuIcon" alt="menu icon"></li>
+                    <li id="open" ><img src="./images/menu.svg" class="menuIcon" alt="menu icon"></li>
                 </ul>
```

Comment:
- [ ] Your Design is very amazing 😃 and the animation is so beautiful🎉, but I noticed something here at the burger icon I see that there is a bullet from the li, so please remove this bullet by giving the li
 ```list-style-type: none``` and with that, it will be nice for the visitors and be like the Figma Design👌

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/215598878-203e9e85-6793-478f-888f-9ffd3c5bd755.png)   |    ![image](https://user-images.githubusercontent.com/83360636/215598842-4f63ec27-0cb7-48ec-8611-8a9cd72f6a25.png)  |